test/ssltestlib.c

```diff
@@ -17,18 +17,28 @@
 #ifdef OPENSSL_SYS_UNIX
 # include <unistd.h>
 
-static ossl_inline void ossl_sleep(unsigned int millis) {
+static ossl_inline void ossl_sleep(unsigned int millis) 
+{
+# ifdef OPENSSL_SYS_VXWORKS
```

Comment:
regarding the proposed file for vxworks-only stuff: i think we are not at a point where we have critical mass yet but as soon as we would add more code I definitly agree that there should be a file bundling it.

Reply:
Don't worry about that. `rand_unix.c` is an `#ifdef` mess anyway. So it would be better to do a general refactoring than just take care about VxWorks.